### ISSUE TITLE
Albums: Flag pictures as "sensitive" so that they are blurred by default

### ISSUE THREAD

**Issue by flaki** [photoprism/photoprism#4367]
I would like to have an image toggle to mark certain images as "sensitive" or "hidden".

I, like many others, use my phone for all sorts of things. As a dog owner, for example, I frequently take photos of not-entirely-pleasant, gross things like poop or other things, often for my own record or to show it to their vet. People often take photos of themselves, kids for medical and other similar reasons to present them to doctors and other healthcare practitioners.

I would like a one-stop-shop way to mask/hide/remove these images from all listings, similar to how "low quality" images don't show up in search. Having a toggle to mark an image as "sensitive" could then make sure that image thumbnail shows up as blurred in *all* listing: calendar, albums, tags, search etc.

Here is a mockup:

normal|sensitive flag set
--|--
![a thumbnail of forest strawberries in the PhotoPrism gallery](https://github.com/photoprism/photoprism/assets/2089432/23a6d35e-8b94-4fab-8443-0aecc3b623d5) | ![the earlier thumbnail is heavily blurred, with the word "SENSITIVE" overlaid on top](https://github.com/photoprism/photoprism/assets/2089432/5d62b02a-9167-4ece-93cb-47e9b8b8d4df)

Note that this blurring is fairly easy to do with CSS alone, no need for blurhashes or custom thumbnails:

```css
.card.image {
  filter: blur(16px);
  clip-path: rect(0 0 100% 100%);
}
```

It would be also nice to be able to use this flag to filter these images altogether from certain views.

This last part might be related to #656

I would be happy to work on/contribute this feature.

**Comment by graciousgrey** [comment]
Thank you very much for your feedback! 

This sounds very similar to the private flag we do already have implemented (except for the blurring).

 - https://docs.photoprism.app/user-guide/organize/private/

**Comment by lastzero** [comment]
@flaki As I understand the idea, "sensitive" pictures would still be visible to all users in the search results, but blurred by default (so you can e.g. click on a button/icon to display them without blurring)?

**Comment by flaki** [comment]
Yes, what I ended up doing is marking these pictures private for now, however I think it might be worth considering this as an "in-between" state.

I do not have an immediate reason why one wouldn't just use private photos instead, perhaps sometimes one would prefer that these images do appear in context (location- or date-based albums), but want to avoid accidental shock/triggering?

**Comment by dexif** [comment]
Great idea. Wanted to create an issue but it already exists :)
